### ISSUE TITLE
esbuild is not a function

### ISSUE THREAD

**Issue by Tahul** [unjs/unbuild#23]
If you generate a new repository using unbuild, and some parts of your build relies on `rollup`, you get a TypeError breaking the build.

Trace:
```
$ unbuild
ℹ  Building unbuild-issue
Error building /Users/tahul/Projets/issues/unbuild-issue: TypeError: esbuild is not a function
TypeError: esbuild is not a function
    at getRollupOptions (file:///Users/tahul/Projets/issues/unbuild-issue/node_modules/unbuild/dist/chunks/build.mjs:231:7)
    at rollupBuild (file:///Users/tahul/Projets/issues/unbuild-issue/node_modules/unbuild/dist/chunks/build.mjs:141:25)
    at build (file:///Users/tahul/Projets/issues/unbuild-issue/node_modules/unbuild/dist/chunks/build.mjs:359:9)
    at async main (file:///Users/tahul/Projets/issues/unbuild-issue/node_modules/unbuild/dist/cli.mjs:31:3)
```

Reproduction: https://github.com/Tahul/unbuild-issue

Thanks for your help 🙏

**Comment by Atinux** [comment]
I can confirm on https://glitch.com/edit/#!/misty-universal-outrigger

Open Terminal > `npm run build`

**Comment by pi0** [comment]
Fixed in unbuild@0.5.12
